zkevm-circuits/src/evm_circuit/param.rs

```diff
@@ -27,7 +27,7 @@ pub(crate) const N_COPY_COLUMNS: usize = 2;
 // Number of copy columns for phase2
 pub(crate) const N_PHASE2_COPY_COLUMNS: usize = 1;
 
-pub(crate) const N_BYTE_LOOKUPS: usize = 24;
+pub(crate) const N_BYTE_LOOKUPS: usize = 25;
```

Comment:
@lispc Without replacing `24` with `25`, the all test cases of `BeginTxGadget` failed to run. Could you help confirm if it is right fix? Thanks.

Reply:
After discussing with @lispc , it could be fixed to `25`. Thanks.